docs/src/components/ComponentDoc/ComponentExamples.tsx

```diff
@@ -82,23 +81,24 @@ export default class ComponentExamples extends React.Component<ComponentExamples
   )
 
   private testExamplesStructure(displayName: string, allPaths: string[]): string[] {
```

Comment:
given the return value of this method, it seems to be a good idea to rename it to something that is more descriptive, like `getMissingExamplePaths` - as otherwise it is completely unclear what this method is supposed to return